### PR TITLE
bump htslib/samtools/bcftools

### PR DESCRIPTION
**Pull request by matthdsm**
Mega bump of samtools/bcftools/htslib across all modules

Awaiting https://github.com/BioContainers/multi-package-containers/pull/3071 for the new multi package containers

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by sateeshperi** [comment]
don't you have to update snapshots as well ?

**Comment by matthdsm** [comment]
> don't you have to update snapshots as well ?

Yes I do, this PR isn't finished yet

**Comment by sateeshperi** [comment]
my bad...plz ignore 😁

**Comment by mashehu** [comment]
wait, did I miss an nf-core hackathon about to happen soon? 😜

File: modules/nf-core/angsd/gl/environment.yml
Comment (maxulysse) [inline modules/nf-core/angsd/gl/environment.yml:1]: I don't think one should update ant environment.yml without updating the main.nf

Comment (edmundmiller) [reply modules/nf-core/angsd/gl/environment.yml:1]: Unless they're out of sync

**Pull request closed without merging**